measures/BuildExistingModel/measure.rb

```diff
@@ -91,16 +86,6 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDescription('This numeric field should contain the calendar year that determines the start day of week. If you are running simulations using AMY weather files, the value entered for calendar year will not be used; it will be overridden by the actual year found in the AMY weather file.')
     args << arg
 
-    arg = OpenStudio::Measure::OSArgument.makeBoolArgument('debug', false)
-    arg.setDisplayName('Debug Mode?')
-    arg.setDescription('If true: 1) Writes in.osm file, 2) Generates additional log output, and 3) Creates all EnergyPlus output files.')
-    args << arg
-
-    arg = OpenStudio::Measure::OSArgument.makeBoolArgument('add_component_loads', false)
-    arg.setDisplayName('Annual Component Loads?')
-    arg.setDescription('If true, output the annual component loads.')
-    args << arg
-
```

Comment:
These two arguments removed here since they are applicable to `HPXMLtoOpenStudio`.